src/rgw/rgw_rados.h

```diff
@@ -481,8 +488,10 @@ class RGWRados
   void cls_obj_check_mtime(librados::ObjectOperation& op, const real_time& mtime, bool high_precision_time, RGWCheckMTimeType type);
 protected:
   CephContext *cct;
+  ceph::async::io_context_pool ctxpool{2};
```

Comment:
it looks like calling this `io_context_pool(std::int16_t threadcnt)` constructor will prevent the later call to `start(rgw_asio_thread_count)` from changing the thread count. this should probably just use the default constructor?

Reply:
Fixed